scripts/create_and_deploy_recipe.sh

```diff
@@ -107,7 +114,10 @@ ${GOPATH}/bin/netgoal build -r "${ROOTDIR}" -n "${NETWORK}" --recipe "${RECIPEFI
 export S3_RELEASE_BUCKET="${S3_RELEASE_BUCKET}"
 ${SRCPATH}/scripts/upload_config.sh "${ROOTDIR}" "${CHANNEL}"
 
-if [ "${NO_DEPLOY}" = "" ]; then
-    # Now generate a private build using our custom genesis.json and deploy it to S3 also
-    ${SRCPATH}/scripts/deploy_private_version.sh -c "${CHANNEL}" -f "${ROOTDIR}/genesisdata/genesis.json" -n "${NETWORK}" -b "${S3_RELEASE_BUCKET}"
+# Build and Deploy binaries
+if [[ ${SKIP_BUILD_DEPLOY} != "true" ]]; then
```

Comment:
when comparing bash variables, you need to ensure placing them in double quotes to avoid it being rendered empty causing the script to fail.

Reply:
added double quotes around variable.